app/serializers/search_results/base_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class SearchResults::BaseSerializer < ActiveModel::Serializer
```

Comment:
This base serializer can be extended to include extra attributes from the model for a discussion, motion, or comment.